Automated vuln-db publishing has broken

I got an alert that our trivy vulndb wasn't fresh anymore. Seems like the last release was three days ago, when they'd normally be every 12 hours.

![image](https://user-images.githubusercontent.com/3473798/102499495-2882a780-4073-11eb-9134-04345cde245e.png)

Looking at the github actions history, it looks like they've just [started failing](https://github.com/aquasecurity/trivy-db/runs/1554027650?check_suite_focus=true): 

```
Run make db-fetch
make: *** No rule to make target 'db-fetch'.  Stop.
Error: Process completed with exit code 2.
```

![image](https://user-images.githubusercontent.com/3473798/102499614-4f40de00-4073-11eb-932d-125c60480539.png)


It seems the `db-fetch` Makefile target was changed in @simar7's PR here: https://github.com/aquasecurity/trivy-db/pull/65/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L60

Sounds like we probably want to update the github actions script to use the new `make db-fetch-langs` and possibly `db-fetch-vuln-list-master` too??

--- a/.github/workflows/cron.yml
+++ b/.github/workflows/cron.yml
@@ -21,7 +21,7 @@ jobs:
       uses: actions/checkout@v1
 
     - name: Download vuln-list and advisories
-      run: make db-fetch
+      run: make db-fetch-langs db-fetch-vuln-list-main
 
     - name: Build the binary
       run: make build

